minicluster/src/main/java/tachyon/master/LocalTachyonCluster.java

```diff
@@ -157,7 +159,7 @@ private void mkdir(String path) throws IOException {
     }
   }
 
-  public void start() throws IOException {
+  private void init() throws IOException {
```

Comment:
It is necessary to change this code. Currently I can't set the configuration for tachyon cluster before it started. Take setting the authentication mechanism for example, when I want to set this configuration, I must at firstly get the tachyonConf insatnce. But the tachyonConf instance was created in the start method. That's the reason I changed this part source code